silver/models.py

```diff
@@ -974,9 +970,7 @@ def add_total_value_to_document(self, billing_date, invoice=None,
                         if bsd <= self.cancel_date <= bed:
                             bed = self.cancel_date
 
-                    if (self.state == self.STATES.ACTIVE or
-                        self.cancel_date >= self.trial_end
-                    ):
+                    if (self.state == self.STATES.ACTIVE or self.cancel_date >= self.trial_end):
```

Comment:
More than 80 characters on a line. There are more lines with exceeding characters in this file.